packages/server-wallet/src/objectives/objective-manager.ts

```diff
@@ -103,19 +101,17 @@ export class ObjectiveManager {
               await this.chainService.concludeAndWithdraw([protocolState.app.supported!]);
               return;
             case 'RequestLedgerFunding': {
-              const ledgerChannelId = await determineWhichLedgerToUse(protocolState.app, tx);
               await LedgerRequest.requestLedgerFunding(
                 protocolState.app.channelId,
-                ledgerChannelId,
+                protocolState.app.fundingLedgerChannelId as string,
```

Comment:
Why does this need type assertion? The type system should be narrowing the type of `fundingLedgerChannelId` for us.

Reply:
It could be `undefined` because not _all_ `ChannelState` objects have this value and we do not have any type narrowing to detect it (yet).